cluster/clean.sh

```diff
@@ -2,9 +2,9 @@
 
 echo 'Cleaning up ...'
 
-./cluster/kubectl.sh delete --ignore-not-found -f deploy/cluster-network-addons-operator_01_crd.yaml
-./cluster/kubectl.sh delete --ignore-not-found -f deploy/cluster-network-addons-operator_02_rbac.yaml
-./cluster/kubectl.sh delete --ignore-not-found -f deploy/cluster-network-addons-operator_00_namespace.yaml
+./cluster/kubectl.sh delete --ignore-not-found -f _out/namespace.yaml
```

Comment:
I think we need to first remove the network-addons-config maybe wait so time for all the related object (multus, macpool, linux bridge, sriov) to be remove then the operator and then the namespace?

Reply:
You are not wrong. This functionality is not implemented yet (we need to add finalizer that will wait with removal of NetworkAddonsConfig until all components are removed). However, it is not a part of this PR.

Reply:
its ok if we don't wait on this PR but this way we remove the namespace(and the operator inside of it) I think for now we can just remove the CR wait for like 5-10 second so the operator can delete the render manifest and then remove the operator without waiting for all the components to be removed and implement the finalizer in a separate PR

Reply:
I changed it so order of manifest removals is the same as it was before this PR, hence there is no change related to this PR other than changing location of those manifests.

Synchronization is not a top priority now.